com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineGlobalSettings.cs

```diff
@@ -99,11 +99,15 @@ static internal HDRenderPipelineGlobalSettings Ensure(bool canCreateNewAsset = t
             }
             Debug.Assert(assetCreated, "Could not create HDRP's Global Settings - HDRP may not work correctly - Open the Graphics Window for additional help.");
             UpdateGraphicsSettings(assetCreated);
+#else
+            if (HDRenderPipelineGlobalSettings.instance == null)
+            {
+                Debug.LogError("Cannot find any HDRP Global Settings asset. Make sure to include one in your Player.");
+            }
```

Comment:
So we do not atempt to migrate in runtime. Seams ok but should we force migration in prebuild then ?

Reply:
yes i actually was wondering this. i was thinking adding a warning for now and discuss the build workflow with you afterwards. i can fix the workflow at the same time if necessary